pkg/test/framework/environments/kubernetes/settings.go

```diff
@@ -45,6 +46,12 @@ const (
 
 	// LatestTag value
 	LatestTag = "latest"
+
+	// DefaultDeployTimeout for Istio
+	DefaultDeployTimeout = time.Second * 480
+
+	// DefaultUndeployTimeout for Istio.
+	DefaultUndeployTimeout = time.Second * 600
```

Comment:
More readable to use time.Minute?

Reply:
I considered that, but this is what they're using in the old framework.